sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -50,6 +50,7 @@ enum {
 #define SOF_SDW_PCH_DMIC		BIT(6)
 #define SOF_SSP_PORT(x)		(((x) & GENMASK(5, 0)) << 7)
 #define SOF_SSP_GET_PORT(quirk)	(((quirk) >> 7) & GENMASK(5, 0))
+/* Deprecated and no longer supported by the code */
 #define SOF_SDW_NO_AGGREGATION		BIT(14)
```

Comment:
How about remove `#define SOF_SDW_NO_AGGREGATION		BIT(14)`?

Reply:
Yeah can remove it completely if you want, I was just a little concerned about a quirk being re-used in the future could result in weird behaviour when moving between older and newer kernel versions.

Reply:
It's fine to keep as deprecated, we are not under too much pressure in terms of bits used by quirks. We can always reclaim this later.